resources.tex

```diff
@@ -38,15 +40,20 @@ \subsection{Quotas and allocation} \label{sec:quotas}
 This however needs more work.
 
 
-%{\bf In operations should we use all Google Cores for interactive batch Dask/Spark ? Send all PanDA to USDF ? But then we do not have 10\% there since we spent on Google.} - PROBALBY NOT FEASABLE
+In operations we propose to use all Google Cores for interactive batch Dask/Spark.
```

Comment:
The use of "batch" in both this and the next sentence seems to obscure the point I think you are trying to make: catalog processing with Dask/Spark goes to Google; any processing (images or, less often, catalogs) done with BPS goes to USDF batch.

Reply:
you could use dask to procces a few images right ? I guess you dont gain much ..

Reply:
i will add your wording